src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -483,8 +499,6 @@
 
     public static final Key<MutableBoundedValue<Float>> HEIGHT = KeyFactory.fake("HEIGHT");
 
-    public static final Key<MutableBoundedValue<Integer>> HELD_EXPERIENCE = KeyFactory.fake("HELD_EXPERIENCE");
```

Comment:
HELD_EXPERIENCE should be deprecated not removed, and then it'll be removed in the next major version bump.